neo/IO/Helper.cs

```diff
@@ -231,6 +265,23 @@ public static void WriteVarBytes(this BinaryWriter writer, byte[] value)
             writer.Write(value);
         }
 
+        public static void WriteArrayVarBytesArray(this BinaryWriter writer, byte[][] values)
```

Comment:
`WriteVarBytesArray`

Reply:
Fixed. I must have typo'd fixing that earlier.